src/app/services/user/user.service.ts

```diff
@@ -126,12 +110,12 @@ export class UserService extends RequestService {
    */
   public async resetPassword(data: ResetPasswordRequest): Promise<ForgotPasswordResponse | ErrorResponse> {
     try {
-      return await lastValueFrom(this.httpClient.post<ForgotPasswordResponse>(this.BASE_URL + '/reset-password/', data))
+      return await lastValueFrom(this.httpClient.post<ForgotPasswordResponse>(this.BASE_URL + '/auth' + '/reset-password/', data));
```

Comment:
A rota de alterar senha mudou ?

Se sim, pode concatenar direto `this.BASE_URL + '/auth/reset-password/'`